src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -236,17 +261,32 @@ export class TaskListWrapperComponent implements OnInit {
   private doLoad(): void {
     this.showSpinner$ = this.loadingService.isLoading;
     const loadingToken = this.loadingService.register();
+    this.isPaginationEnabled$.subscribe(enabled => {
```

Comment:
Do we need to have nested subscribes here?. Can we utilise mergeMap. 
  `this.isPaginationEnabled$. pipe(mergeMap((enabled) => this. performSearchPagination() || this. performSearch()).subscribe()`